balancer-js/src/modules/pools/pool-types/concerns/stable/join.concern.integration.spec.ts

```diff
@@ -4,34 +4,33 @@ import { expect } from 'chai';
 import {
   BalancerError,
   BalancerErrorCode,
```

Comment:
Similar to the latest ComposableStable test can we add tests for:
`should join using encoded data - single token has value`
`should return correct attributes for joining`

Reply:
@lgahdl this is still not done right?